spec/features/option_set_flow_spec.rb

```diff
@@ -52,6 +52,11 @@
     add_options(%w[Pear])
     click_button("Save")
     expect(page).to have_selector("td.options_col div", text: "Banana, Apple, Pear")
+
+    # Test export page does not error out
```

Comment:
Would be good to add 'export' to the description of the scenario as well, e.g. "creating, showing, editing, and exporting"

Reply:
oooh thanks for the catcg